src/backend/distributed/commands/multi_copy.c

```diff
@@ -471,6 +471,15 @@ CopyToExistingShards(CopyStmt *copyStatement, char *completionTag)
 				fmgr_info(textSendAsJsonbFunctionId,
 						  &copyDest->columnOutputFunctions[columnIndex]);
 			}
+			else
+			{
+				List *nameList = list_make2(makeString("pg_catalog"),
+											makeString("textout"));
+				Oid paramOids[1] = { TEXTOID };
+				Oid textoutFunctionId = LookupFuncName(nameList, 1, paramOids, false);
```

Comment:
We could cache this, but I think it is not necessary, since a COPY command is supposed to run for a much longer time than fetching an OID.